addons/languages/icelandic/pack/src/main/res/values-is/icelandic_pack_strings_dont_translate.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools"
+           xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2"
+           tools:ignore="MissingTranslation,TypographyDashes">
+
+    <string name="icelandic_dictionary">Íslenska</string>
+
+    <string name="is_keyboard">Íslenska</string>
+    <string name="is_keyboard_qwerty">Íslenska QWERTY</string>
```

Comment:
Seems to be unused: https://github.com/AnySoftKeyboard/AnySoftKeyboard/runs/6773630658?check_suite_focus=true#step:6:9281

Reply:
string ressource removed